lib/naughty/core.lua

```diff
@@ -66,11 +66,19 @@ notifications, e.g.
 naughty.config = {
     padding = dpi(4),
     spacing = dpi(1),
-    icon_dirs = { "/usr/share/pixmaps/", },
+    icon_dirs = { "/usr/share/icons/hicolor/", "/usr/share/icons/", "/usr/share/pixmaps/", },
     icon_formats = { "png", "gif" },
     notify_callback = nil,
 }
 
+if beautiful.icon_theme then
+    local theme = beautiful.icon_theme
+    local theme_dir = "/usr/share/icons/" .. theme .. "/"
+    table.insert(naughty.config.icon_dirs, 1, theme_dir)
+    table.insert(naughty.config.icon_dirs, 1, theme_dir .. "base/")
+    table.insert(naughty.config.icon_dirs, 1, theme_dir .. "symbolic/")
+end
```

Comment:
As Travis points out, this does not work: https://travis-ci.org/awesomeWM/awesome/jobs/252054577
```
error: lib/beautiful/init.lua:33: May not access theme before beautiful.init()
```
This code runs during `require()` and `beautiful.init` is only called later (look at the default config). Thus, `beautiful.icon_theme` will always be `nil` when this code runs.